api/python/quilt3/packages.py

```diff
@@ -857,6 +863,12 @@ def set_meta(self, meta):
         self._meta['user_meta'] = meta
         return self
 
+    def get_meta(self):
+        """
+        Gets user metadata on this Package.
+        """
+        return self.meta or None
```

Comment:
I don't think this is correct because look [what `set_meta` does](https://github.com/quiltdata/quilt/blob/f476bf61b08c83e4541c9d234973ae82d99307eb/api/python/quilt3/packages.py#L853). Also, `or None` doesn't buy us anything? If you want to safely get a value from a `dict` use `.get`.

Reply:
Also. `self.meta` is not defined anywhere?

Reply:
Package and PackageEntry classes already have a property [`meta`](https://github.com/quiltdata/quilt/blob/f476bf61b08c83e4541c9d234973ae82d99307eb/api/python/quilt3/packages.py#L381) defined which returns the user metadata or an empty dict.

Reply:
True but look at how it's used. We want to extract the `user_meta` portion. Also, we shouldn't have to do anything with PackageEntry as I mentioned below.